app/addons/cors/components.js

```diff
@@ -334,6 +334,7 @@ var CORSController = React.createClass({
             <br />
             <button
               type="button"
+              id="bt-enable-disable-cors"
```

Comment:
It makes it easier for the tests so I can use `wrapper.find('#bt-enable-disable-cors')`. Otherwise I either have to query all buttons and get it using an index, or query based on where it is. I find both options make the test very dependent on the HTML structure.